examples/flax/token-classification/requirements.txt

```
@@ -0,0 +1,5 @@
+datasets >= 1.8.0
+jax>=0.2.8
+jaxlib>=0.1.59
+flax>=0.3.4
+optax>=0.0.8
```

```suggestion
optax>=0.0.8
seqeval
```

required for metrics

---

updated